tensorflow/lite/experimental/c/c_api_types.h

```diff
@@ -1 +1 @@
-../../c/c_api_internal.h
\ No newline at end of file
+#include "../../c/c_api_internal.h"
```

Comment:
I'm not sure this will work, is it 100% necessary to build? Can you exclude it from the srcs?

Reply:
Yes, do you mean that need to change following files?

```
experimental/c/BUILD|43 col 10| "c_api_types.h",
experimental/c/BUILD|61 col 10| "c_api_types.h",
experimental/c/c_api.cc|21 col 42| #include "tensorflow/lite/experimental/c/c_api_types.h"
experimental/c/c_api.h|24 col 11| #include "c_api_types.h"
experimental/ios/BUILD.apple|24 col 43| "//tensorflow/lite/experimental/c:c_api_types.h",
experimental/objc/TensorFlowLiteObjC.podspec|31 col 31| tfl_dir + 'experimental/c/c_api_types.h',
```

Reply:
Hmm, I need to think a bit more about this one, as it will likely fail during our internal submit.

If you don't mind, can we drop the change to this file, and keep the other changes? I can approve that, and get back to you about a fix for this file.